src/main/java/ru/yandex/qatools/allure/command/ReportGenerate.java

```diff
@@ -89,4 +89,21 @@ protected String getLoggerConfigurationArgument() {
         return String.format("-Dorg.slf4j.simpleLogger.defaultLogLevel=%s",
                 isQuiet() || !isVerbose() ? "error" : "debug");
     }
+
+    private String getClassPathArgument() {
+        String version = getConfig().getCurrentVersion();
+        String separator = System.getProperty("path.separator");
+        Path configPath = getConfigPath().getParent().toAbsolutePath();
+        Path bundleJarPath = getExecutablePath(version);
+        String pluginsPath = getPluginsPath().toAbsolutePath() + "/*";
```

Comment:
Do you see something I don't see?
Yes, tested on Windows. Windows understands `/` as path separators.